### PR TITLE
feat: initial files with setup and plugin

### PR DESCRIPTION
**Pull request by shreyanshs7**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+with open("README.md", "r") as f:
+    long_description = f.read()
+
+setup(
+    name="kwargs-enforcer",
+    version=Plugin.version,
+    author="Siva Narayanan",
+    author_email="siva@fyle.in",
```
Comment (sanskarsharma) [inline setup.py]: use fylehq.com

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    description="Flake8 plugin for enforcing kwargs only argument definition.",
+    license="MIT",
```
Comment (sanskarsharma) [inline setup.py:14]: add the licence file

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+from setuptools import setup, find_packages
+from plugin.checker import Plugin
+
+
+with open("README.md", "r") as f:
```
Comment (sanskarsharma) [inline setup.py:5]: README is empty, pls complete

**Review by sanskarsharma — APPROVED** [review]

File: setup.py (hunk 3 above)
Comment (shreyanshs7) [reply setup.py:5]: will add this later

**Pull request merged**